src/js/index.jsx

```diff
@@ -18,19 +18,29 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Store } from 'webext-redux';
 import { Provider } from 'react-redux';
-import App from './apps/App';
+import App from './devtools/App';
 import '../sass/style.scss';
-import { REDUX_PORT, PANEL_NAME } from './helpers/constants';
-
-const store = new Store({ portName: REDUX_PORT });
+import {
```

Comment:
I would consider adding some scoping here, for example place GET_CURRENT_TAB_INFO in messages object